src/NServiceBus.NHibernate/Outbox/OutboxPersister.cs

```diff
@@ -19,9 +20,9 @@ public bool TryGet(string messageId, out OutboxMessage message)
 
             message = null;
 
-            using (var session = StorageSessionProvider.OpenStatelessSession())
+            using (var session = SessionFactoryProvider.SessionFactory.OpenStatelessSession())
             {
-                using (var tx = session.BeginAmbientTransactionAware(IsolationLevel.ReadCommitted))
+                using (var tx = session.BeginTransaction(IsolationLevel.ReadCommitted))
```

Comment:
You're right, I think we should explicitly suppress here. Even better, we should suppress in the core (`OutboxDeduplicationBehavior`)